clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -163,7 +207,26 @@ protected ClusteringServiceConfiguration(ClusteringServiceConfiguration baseConf
    * @return the cluster {@code URI}
    */
   public URI getClusterUri() {
-    return clusterUri;
+    return connectionSource.getClusterUri();
+  }
+
+  /**
+   * The {@code ConnectionSource} of the cluster, containing either a {@code URI}, or an {@code Iterable<InetSocketAddress>}
+   * of the servers in the cluster.
+   *
+   * @return a cluster {@code ConnectionSource}
+   */
+  public ConnectionSource getConnectionSource() {
+    return connectionSource;
+  }
+
+  /**
+   * Cache manager name for the cache.
+   *
+   * @return the cache manager name
+   */
+  public String getCacheManagerName() {
```

Comment:
This shouldn't need to be exposed here since you already expose `ConnectionSource`

Reply:
Indeed. Will remove.